### PR TITLE
53 our mission component

### LINKED ISSUE #53: Our Mission Component - <Landing Page>
* Signed in only: `false`
* Admin only:	`false`

image

### PR DESCRIPTION
**Pull request by ghost**
## Result
* Desktop: 
image
* Moblie
image

### CODE REVIEW

**Review by haniolwan — APPROVED** [review]

File: client/src/App.css [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+}
+
+a {
+  text-decoration: none;
+}
+
+button {
+  cursor: pointer;
+  border: none;
+}
```
Comment (Raghad-Mezied) [inline client/src/App.css:22]: All these general styles should be in the theme if it's not already controlled by the components

File: client/src/components/OurMission/OurMission.jsx [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+        <p className="our-mission-description-title">Our Mission</p>
+        <p className="our-mission-description-paragraph">
+          We At Our Company name, we give the best services for users and for
+          caregivers to donate their extra money so that we steal the money from
+          them.
+        </p>
+      </div>
+    </section>
+  );
+}
```
Comment (Raghad-Mezied) [inline client/src/components/OurMission/OurMission.jsx:20]: Check antd components

File: client/src/components/index.jsx [hunk 3]
```diff
@@ -0,0 +1,2 @@
+// eslint-disable-next-line import/prefer-default-export
+export { default as OurMission } from './OurMission/OurMission';
```
Comment (Raghad-Mezied) [inline client/src/components/index.jsx:2]: Edit the import and fix the eslint

**Review by Raghad-Mezied — CHANGES_REQUESTED** [review]

**Review by haniolwan — APPROVED** [review]

File: client/src/components/OurMission/index.jsx [hunk 4]
```diff
@@ -0,0 +1,25 @@
...
+  return (
+    <section className="our-mission-section">
+      <div className="our-mission-image-container">
+        <div className="green-background" />
+      </div>
+      <div className="our-mission-description-container">
+        <Paragraph className="title">Our Mission</Paragraph>
+        <Paragraph className="description">
+          We At Our Company name, we give the best services for users and for
+          caregivers to donate their extra money so that we steal the money from
```
Comment (HaniOlwan) [inline client/src/components/OurMission/index.jsx:17]: put this as a description 
`Bless others with your donation. Our mission is to empower underserved communities and provide the best life for people in need.`

**Review by Raghad-Mezied — APPROVED** [review]

**Review by farah2003 — APPROVED** [review]

**Review by DinaAlghazali — APPROVED** [review]

**Pull request merged**